### ISSUE TITLE
Fix stories background color not working in the docs tab

### ISSUE THREAD

**Issue by alihamoody** [gymnasium/gym-ui#109]
Example:
![image](https://user-images.githubusercontent.com/49724761/65978438-b314c200-e441-11e9-8235-5d70f3f898d3.png)

**Comment by alihamoody** [comment]
This issue is not related to converting story format to CSF. It is because the `backgrounds` parameter is no longer supported and now replaced by `theme`

**Comment by alihamoody** [comment]
The solution is to switch to dark theme when viewing components that require dark background. Unfortunately, this makes the whole storybook switch to dark mode on the particular story.

I tried to format the **content**  part with dark theme background but wasn't practical. Here are the reaons:
1. It is possible to switch the story content-only to dark background using theme.
2. Then, we need to change the foreground (text) color to white or light.
3. Unfortunately, the left navigation text color changes as well. It is not decoupled from the story content setting.
4. Once that happens, we have to change the navigation panel to dark background, which means switching the whole app to dark mode.

So, I resorted to simply switching to the theme to dark instead.
